NextcloudTalk/SettingsTableViewController.swift

```diff
@@ -0,0 +1,705 @@
+        if !NCDatabaseManager.sharedInstance().inactiveAccounts().isEmpty {
+            sections.append(SettingsSection.kSettingsSectionAccounts.rawValue)
+        }
+        // Configuration section
+        sections.append(SettingsSection.kSettingsSectionConfiguration.rawValue)
+        // About section
+        sections.append(SettingsSection.kSettingsSectionAbout.rawValue)
+        return sections
+    }
+
+    func getConfigurationSectionOptions() -> [Int] {
+        var options = [Int]()
+        // Video quality
+        options.append(ConfigurationSectionOption.kConfigurationSectionOptionVideo.rawValue)
+        // Open links in
+        if NCSettingsController.sharedInstance().supportedBrowsers.count > 1 {
+            options.append(ConfigurationSectionOption.kConfigurationSectionOptionBrowser.rawValue)
+        }
+        // Read status privacy setting
+        if NCDatabaseManager.sharedInstance().serverHasTalkCapability(kCapabilityChatReadStatus) {
+            options.append(ConfigurationSectionOption.kConfigurationSectionOptionReadStatus.rawValue)
+        }
+        // Contacts sync
+        if NCDatabaseManager.sharedInstance().serverHasTalkCapability(kCapabilityPhonebookSearch) {
+            options.append(ConfigurationSectionOption.kConfigurationSectionOptionContactsSync.rawValue)
+        }
+        return options
+    }
+
+    func getSectionForSettingsSection(section: SettingsSection) -> Int {
+        let sectionNumber = getSettingsSections().firstIndex(of: section.rawValue)
+        if NSNotFound != sectionNumber {
+            return sectionNumber!
+        }
+        return 0
+    }
+
+    func getIndexPathForConfigurationOption(option: ConfigurationSectionOption) -> IndexPath {
+        let section: Int = getSectionForSettingsSection(section: SettingsSection.kSettingsSectionConfiguration)
+        var optionIndexPath: IndexPath = IndexPath(row: 0, section: section)
+
+        let optionRow = getConfigurationSectionOptions().firstIndex(of: option.rawValue)
+
+        if NSNotFound != optionRow {
+            optionIndexPath = IndexPath(row: optionRow!, section: section)
+        }
+        return optionIndexPath
+    }
+
+    // MARK: User Profile
+
+    func refreshUserProfile() {
+        NCSettingsController.sharedInstance().getUserProfile { _ in
+            self.tableView.reloadData()
+        }
+        self.getActiveUserStatus()
+    }
+
+    func getActiveUserStatus() {
+        let activeAccount = NCDatabaseManager.sharedInstance().activeAccount()
+        NCAPIController.sharedInstance().getUserStatus(for: activeAccount) { userStatus, error in
+            if error == nil && userStatus != nil {
+                self.activeUserStatus = NCUserStatus(dictionary: userStatus!)
+                self.tableView.reloadData()
+            }
+        }
+    }
+
+    // MARK: Notifications
+
+    @objc func appStateHasChanged(notification: NSNotification) {
+        let appState = notification.userInfo!["appState"]
```

Comment:
```suggestion
        let appState = notification.userInfo?["appState"]
```
If you safely unwrap it later anyway, there's no need to force unwrap here :)

Reply:
sure :D